Makefile

```diff
@@ -49,6 +50,7 @@ zip:
 	cp -r lambdas/utils lambdas/package_$(lambda_name)
 	cp -r lambdas/models lambdas/package_$(lambda_name)
 	cp -r lambdas/services lambdas/package_$(lambda_name)
+	cp -r lambdas/enums lambdas/package_$(lambda_name)
```

Comment:
This folder needs to be added to the zip make function so that it's included in deployments.

Reply:
We probably also want to include the check for changes in this folder in the dev CI action so that if there is a change to this folder, all Lambdas and built and pushed.

Reply:
Spoke to Scott, this is fine. Resolved.